lib/veritech-server/src/config.rs

```diff
@@ -52,6 +55,8 @@ pub struct Config {
 
     cyclone_spec: CycloneSpec,
 
+    decryption_key_path: CanonicalFile,
```

Comment:
These aren't always paths. We pass them in as base64 strings in prod. SDF Server takes these in using the CryptoConfig object. Can we do that here, too? It will let us keep using our [generated config file](https://github.com/systeminit/si/blob/main/component/init/configs/service.toml).

```
#[builder(default = "CryptoConfig::default()")]
    crypto: CryptoConfig,
```